### ISSUE TITLE
Comma-first vars should be encouraged.

### ISSUE THREAD

**Issue by programmin1** [rwaldron/idiomatic.js#195]
It says "Any project that cites this document as its base style guide will not accept comma first code formatting", yet I have found comma-first is the best way if you don't want multiple var statements

For example:
```var a=1,
      b=3
      c=4
```
will easily make a global variable c, which is likely not intended.

This style:
```var a=1
,   b=3
,   c=4
```
is more obvious when a comma is missing - causing a global variable to be created.

**Comment by acconrad** [comment]
While I agree that it is clearer and easier to find missing commas, this is not a universally accepted axiom of the JS community. Plus, there are even differences in opinion within the pro-comma group. I'd argue that the odd spacing would be cumbersome and just easier to write:

```javascript
var a = 1
    , b = 2
    , c = 3
```

and I'd include spaces between `=` as well.

So as much as I like comma-first, I can't say that as a whole Idiomatic should be evangelizing such a polarizing style.

**Comment by leonardolessa** [comment]
Well, in my case I believe both are bad options, I think multiple var statements is the way to go, [Ben Alman post](http://benalman.com/news/2012/05/multiple-var-statements-javascript/) cleared my mind. And like him, I've never heard a clear case that a single var statement is better than multiple despite being "pretty".

**Comment by LiamKenneth** [comment]
I agree with @leonardolessa, here is a good article about [not using single var](https://danhough.com/blog/single-var-pattern-rant/).

**Comment by stegano** [comment]
I agree with @leonardolessa 2, because programmer is not minimizer&optimizer
